tempest/api/workloadmgr/multiattach_volumes/test_multiattach_volumes.py

```diff
@@ -0,0 +1,631 @@
+    def test_01_multiattach_volumes(self):
+        try:
+            test_var = "tempest.api.workloadmgr.multiattach_volumes.test_image_booted_"
+            tests = [[test_var + "workload_api", 0],
+                     [test_var + "full_snapshot_api", 0],
+                     [test_var + "incremental_snapshot_api", 0],
+                     # [test_var+"snapshot_mount_api", 0],
+                     # [test_var+"filesearch_api", 0],
+                     [test_var + "selectiverestore_api", 0]]
+            # [test_var+"inplacerestore_api", 0],
+            # [test_var+"oneclickrestore_api", 0]]
+            reporting.add_test_script(tests[0][0])
+            self.kp = self.create_key_pair(tvaultconf.key_pair_name)
+            self.vm_id_1 = self.create_vm(key_pair=self.kp)
+            self.vm_id_2 = self.create_vm(key_pair=self.kp)
+            mount_point = "mount_data_b"
+
+            # find volume_type = multiattach. So that existing multiattach volume type can be used.
+            # Get the volume_type_id
+            vol_type_id = -1
+            for vol in CONF.volume.volume_types:
+                if (vol.lower().find("multiattach") != -1):
+                    vol_type_id = CONF.volume.volume_types[vol]
+                    vol_type_name = vol
+
+            if (vol_type_id == -1):
+                raise Exception("No multiattach volume found to create multiattach volume. Test cannot be continued")
+
+            # Now create volume with derived volume type id...
+            self.volume_id = self.create_volume(
+                volume_type_id=vol_type_id, size=10,volume_cleanup=False)
+
+            LOG.debug("Volume ID: " + str(self.volume_id))
+
+            self.volumes = []
+            self.volumes.append(self.volume_id)
+            # Attach volume to vm...
+            self.attach_volume(self.volume_id, self.vm_id_1,attach_cleanup=False)
+            self.attach_volume(self.volume_id, self.vm_id_2,attach_cleanup=False)
+            LOG.debug("Multiattach Volume attached to vm: " + str(self.vm_id_1) + " and " + str(self.vm_id_2))
+            self.disk_names = ["vdb"]
+
+            vol_vm_1 = self.get_attached_volumes(self.vm_id_1)
+            vol_vm_2 = self.get_attached_volumes(self.vm_id_2)
+            LOG.debug("Voulme o VM 1: " + str(vol_vm_1) + " on VM 2:" + str(vol_vm_2))
+            if vol_vm_1 == vol_vm_2:
+                reporting.add_test_step("Attached Multiattach volume to both Instances", tvaultconf.PASS)
+                reporting.set_test_script_status(tvaultconf.PASS)
+            else:
+                reporting.add_test_step("Attached Multiattach volume to both Instances", tvaultconf.FAIL)
+                reporting.set_test_script_status(tvaultconf.FAIL)
+                raise Exception("Multiattach volume failed to attach existing instance")
+            fip = self.get_floating_ips()
+            LOG.debug("\nAvailable floating ips are {}: \n".format(fip))
+            # if len(fip) < 8:
```

Comment:
Remove this commented line.

Reply:
it will be required later so its commented for now